Content.Server/Power/SMES/SmesComponent.cs

```diff
@@ -5,82 +5,24 @@
 using Robust.Server.GameObjects;
 using Robust.Shared.Timing;
 
-namespace Content.Server.Power.SMES
+namespace Content.Server.Power.SMES;
+
+/// <summary>
+///     Handles the "user-facing" side of the actual SMES object.
+///     This is operations that are specific to the SMES, like UI and visuals.
+///     Logic is handled in <see cref="PowerSmesSystem"/>
+///     Code interfacing with the powernet is handled in <see cref="BatteryStorageComponent"/> and <see cref="BatteryDischargerComponent"/>.
+/// </summary>
+[RegisterComponent, Access(typeof(SmesSystem))]
+public sealed class SmesComponent : Component
```

Comment:
Make every field VVable (ReadWrite if its supported or makes sense) and remove the static field since it makes no sense in components (there are some around in the codebase but its a bad practice)

Reply:
I made them all readonly.  Readwrite didn't make sense to me since they will still be driven/overwritten by the battery component, but let me know if you disagree.